closure/webfiles/web_library.bzl

```diff
@@ -209,6 +209,7 @@ web_library = rule(
     executable = True,
     attrs = {
         "path": attr.string(),
+        "bind": attr.string(default = "0.0.0.0:6006"),
```

Comment:
Let's introduce `host` and `port` attributes; I think it will be more self documenting and in many cases users will only change the port or the host.